### ISSUE TITLE
Different dropout rate between training and testing?

### ISSUE THREAD

**Issue by elwell67** [yaringal/DropoutUncertaintyExps#1]
Any suggestions on how to implement the stochastic predictor with a _different_ dropout rate than that which was used in training?  I have tried to modify the layer attributes (<layer>.rate), but this does not change the output of the stochastic predictor function (built on the keras backend function).
